core/src/avm1/globals/shared_object.rs

```diff
@@ -179,8 +179,8 @@ pub fn get_local<'gc>(
     let mut movie_url = if let Some(url) = movie.url().and_then(|url| url::Url::parse(url).ok()) {
         url
     } else {
-        log::error!("SharedObject::get_local: Unable to parse movie URL");
```

Comment:
(maybe handle "movie.url() is None" and "parsing url failed" cases separately? That would keep the previous check)